patches/components-translate-core-browser-language_state.h.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/components/translate/core/browser/language_state.h b/components/translate/core/browser/language_state.h
+index 3cee8dc8f310..2957ecb16a42 100644
+--- a/components/translate/core/browser/language_state.h
++++ b/components/translate/core/browser/language_state.h
+@@ -70,7 +70,7 @@ class LanguageState {
+   void set_translation_declined(bool value) { translation_declined_ = value; }
+ 
+   // Whether the translate is enabled.
+-  bool translate_enabled() const { return translate_enabled_; }
++  bool translate_enabled() const { return false; }
```

Comment:
Usually we'll subclass and use chromium_src overrides but from what I see in this case it'd lead to more patching than this.